dj_rest_auth/jwt_auth.py

```diff
@@ -1,4 +1,4 @@
-from django.utils import timezone
+from datetime import datetime

```

Comment:
Whats the intent with this change?

Reply:
See the third bullet in my PR description about the `django.utils.timezone` deprecation

Reply:
I'm just now seeing @dan-blanchard comments.